apps/frontend/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "ui",
```

```suggestion
  "name": "@trader-app/ui",
```
Rename package, this is how it shows up in your node_modules so you want to prefix it, usually "@project-name/package-name